test/interface.jl

```diff
@@ -255,12 +255,10 @@ end
                 # FIXME: `SimplexBijector` results in ∞ gradient if not in the domain
                 if !contains(t -> t isa SimplexBijector, b)
                     b_logjac_ad = [logabsdet(ForwardDiff.jacobian(b, xs[:, i]))[1] for i = 1:size(xs, 2)]
-                    tol = isclosedform(b) ? 1e-9 : 1e-1
-                    @test logabsdetjac(b, xs) ≈ b_logjac_ad rtol=tol atol=tol
+                    @test logabsdetjac(b, xs) ≈ b_logjac_ad atol=1e-9
```

Comment:
I noticed that these tests now also pass for planar layers with stricter tolerances. I am not sure how useful `isclosedform` actually is anymore?

Reply:
Hmm, maybe just leave it for now and open an issue? Might be best to remove in a different PR if so + by default it returns `true` for all bijectors.

Reply:
Sure, I don't want to add anything to this PR :slightly_smiling_face: